openapi/templates/elixir/connection.ex.mustache

```diff
@@ -0,0 +1,92 @@
+{{>licenseInfo}}
+defmodule {{moduleName}}.Connection do
+  @moduledoc """
+  Handle Tesla connections for {{moduleName}}.
+  """
+
+  use Tesla
+
+  # Add any middleware here (authentication)
+  plug Tesla.Middleware.BaseUrl, "{{{basePath}}}"
+  plug Tesla.Middleware.Headers, [{"user-agent", "Elixir"}]
```

Comment:
```suggestion
  plug Tesla.Middleware.Headers, [{"user-agent", "Elixir x.x.x"}]
```

Would be useful to add the package version to the user agent

```elixir
:application.get_key(dep, :vsn)
```

Reply:
We no longer use the templates as our changes got merged into the upstream https://github.com/OpenAPITools/openapi-generator project; but I think this would be really good to add there with a PR perhaps 👀

Reply:
I would for sure, the more metadata we have about particular build the better, so I would recommend it